index.md

```diff
@@ -5,11 +5,15 @@
 layout: home
 ---
 
-PELUX is a Linux based platform for your automotive infotainment project. The
-PELUX software architecture is optimized to secure the shortest path from
-silicon to pixel and thereby provide optimal performance to deliver a stunning
-infotainment solution. It's based on [Yocto](http://yoctoproject.org/) and
-offers an improved developer experience.
+The PELUX Base Platform is a Linux based platform used for kickstarting automotive software projects. The
+PELUX Base Platform software architecture and design is optimized to secure the shortest path from
+silicon to pixel and thereby provides optimal patterns to deliver a stunning in-vehicle infotainment solution. 
+Though PELUX is not limited to infotainment systems. It aims to target other converged automotive systems,
+like the advanced driver-assistance systems (ADAS) and telematics units.PELUX leverages well known open 
```

Comment:
Missing space between the period and "PELUX".

Reply:
Thanks!